rules/node.libsonnet

```diff
@@ -33,7 +33,7 @@
               count by (%(clusterLabel)s, node) (sum by (node, cpu) (
                 node_cpu_seconds_total{%(nodeExporterSelector)s}
               * on (namespace, %(podLabel)s) group_left(node)
-                node_namespace_pod:kube_pod_info:
+                topk by(namespace, %(podLabel)s) (1, node_namespace_pod:kube_pod_info:)
```

Comment:
Using `group` here instead of `topk` seems more appropriate, no? We don't really care about the sorting or actual top, just any 1 of the values.

Reply:
not really because `group` would discard the "node" label. After more thinking, we may not need to join `node_cpu_seconds_total` with `node_namespace_pod:kube_pod_info:` since in practice, the node_exporter's "instance" label maps to "node" but I might be missing some context on why we're using a join in the first place?

Reply:
> since in practice, the node_exporter's "instance" label maps to "node"

The idea in the kubernetes-mixin was always to not rely on assumptions like this, which is why the join is used.